collect_app/src/androidTest/java/org/odk/collect/android/support/rules/CallbackCountingTaskExecutorRule.kt

```diff
@@ -0,0 +1,16 @@
+package org.odk.collect.android.support.rules
+
+import androidx.arch.core.executor.testing.CountingTaskExecutorRule
+
+class CallbackCountingTaskExecutorRule : CountingTaskExecutorRule() {
+
+    private var finishedCallback: Runnable? = null
```

Comment:
There is no need to make it nullable I think.

Reply:
As I checked it probably need to be nullable. If I change it to not nullable then some tests are failing.

Reply:
Which test for example?

Reply:
All tests in ConfigureWithQRCodeTest class.

Reply:
Ok now I see.